boards/arm/v2m_beetle/doc/v2m_beetle.rst

```diff
@@ -237,7 +237,7 @@ the :ref:`hello_world` application.
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
    :board: v2m_beetle
-   :goals: build flash
+   :goals: build
```

Comment:
I see you removed "flash" from the goals, but i don't see any replacement text for how to flash?  Perhaps your comment "the right way to flash and debug is under some discussion" means you don't have replacement text yet...

Reply:
Sorry to have been unclear. I tried to explain this change in the commit log:

```
This board doesn't support the 'flash' goal, which was mistakenly
included during the CMake conversion.
```

This is one of the "minor issues around flashing and DT that I noticed while poking around".